Ginger/GingerCoreNET/SourceControl/GITSourceControl.cs

```diff
@@ -482,6 +487,40 @@ public override void Init()
             Console.WriteLine("GITHub - Init");
         }
 
+        public bool IsGitIgnoreFileMissing()
+        {
+            string gitignoreFilePath = Path.Combine(RepositoryRootFolder, ".gitignore");
```

Comment:
Better to add field to maintain git ignore file path so it can be used at all places